### PR TITLE
Add toggle pause to docs

### PR DESCRIPTION
**Pull request by JamesLDeveloper**
Add toggle pause section to docs
image

image

Add tip section informing user that they need to write their language and type in upper case.
image

### CODE REVIEW

File: docs/guide/cli/creating a Function.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ Once you are logged in, you can create a function by following these steps:
 -   Enter one of the available languages `go` `py` `js` `cs` `rb` `rs`
 -   Enter description `optional` `ascii` `max=80`
 
+::::tip tip
+
+When choosing a language make sure to use lowercase (this is different than when creating a function via calls to the API).
+
+::::
+
```
Comment (tomaswarynyca) [inline docs/guide/cli/creating a Function.md:38]: it doesn't make much sense to add this, it is better to make the CLI handle this

Comment (JamesLDeveloper) [reply docs/guide/cli/creating a Function.md:38]: True, but until an update to the CLI is released to fix this issue its best to let people know?

Comment (JamesLDeveloper) [reply docs/guide/cli/creating a Function.md:38]: Then we can just remove this tip from the docs?

Comment (tomaswarynyca) [reply docs/guide/cli/creating a Function.md:38]: The new version is coming out today or Monday so there's not much sense in having it

Comment (JamesLDeveloper) [reply docs/guide/cli/creating a Function.md:38]: Ah ok, will update. Thanks

File: docs/cli/Function.md [hunk 2]
```diff
@@ -66,6 +66,12 @@ This subcommand of the function command initiates the configuration process of c
 
 ::::
 
+::::tip tip
+
+When choosing your language make sure to use lowercase (this is different than when creating a function via a call to the API).
+
+::::
+
```
Comment (tomaswarynyca) [inline docs/cli/Function.md:74]:
```suggestion
```

**Pull request merged**